packages/backend/src/openapi/specs/webhooks.yaml

```diff
@@ -358,10 +358,10 @@ components:
         value:
           type: string
           format: uint64
-          description: 'The value is an unsigned 64-bit integer amount, represented as a string.'
+          description: 'An unsigned 64-bit integer amount, represented as a string.'
         assetCode:
           type: string
-          description: The assetCode is a code that indicates the underlying asset. This SHOULD be an ISO4217 currency code.
+          description: A code that indicates the underlying asset. This SHOULD be an ISO4217 currency code.
```

Comment:
Maybe we can mention that it should be `an ISO4217` if possible. Kind of like we have in the docs page for assets:

"The asset code, generally an ISO 4217 currency code where available."